src/uri.jl

```diff
@@ -173,6 +173,7 @@ escape(str::AbstractString, safe::Function=issafe) =
 
 escape(bytes::Vector{UInt8}) = bytes
 escape(v::Number) = escape(string(v))
+escape(v::Symbol) = escape(string(v))
```

Comment:
@quinnj, @KristofferC what do you think about just providing `escape(v) = escape(string(v))` ?

Reply:
Yeah, I thought about the same thing just now. I think it makes sense.

Reply:
One of my users was expecting `escape(::DateTime)` to work  https://github.com/JuliaCloud/AWSSDK.jl/issues/4 .
@quinnj what do you think of adding `escape(v) = escape(string(v))` ?

Reply:
Sure.

Reply:
... ok, so now that I've look at "just doing it", there is a hitch:
`escape(query)` is already defined to work Dicts, Vectors{Pair} etc, anything that acts like an iterator over pairs (`... for (k,v) in query`).
Best to leave it as is until Julia has some kind of "protocols" feature that allows dispatching on "dict-like or sequence of pair-like thing".

Reply:
Let's add a case for `Dates.TimeType` for now then.